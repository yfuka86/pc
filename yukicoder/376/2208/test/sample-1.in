3
1 3 2 6
-1 6 -3 7
2 2 -1000 1
