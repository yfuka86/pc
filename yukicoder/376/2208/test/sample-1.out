12
10
-1999
